app/controllers/application_controller.rb

```diff
@@ -50,17 +50,6 @@ def invalidate_record(uuid)
 
   def user_data_params
     UserDataParams.new(@page_answers).answers
-  rescue ActionController::UnfilteredParameters
```

Comment:
We will see it even without this code, just not with the debug (but the debug was not very helpful at all sadly!) because the exception will be raised if it happens again.